src/selenium/sessionPage.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const sessionPage = Object.create(BasePage);
+
+sessionPage.getSessionTitle = function() {
+  const self = this;
+  const titleId = 'title-1090';
+
+  const sessionPromise = new Promise(function(resolve) {
+    const title = self.find(By.id(titleId)).getText();
+
+    resolve(title);
+  });
+
+  return sessionPromise;
+};
+
+sessionPage.getSessionBackgroundColor = function() {
+  const self = this;
+  const titleId = 'title-1090';
+
+  const sessionPromise = new Promise(function(resolve) {
+    const backgroundColor = self.find(By.id(titleId)).getCssValue('background-color');
+
+    resolve(backgroundColor);
+  });
+
+  return sessionPromise;
+};
+
+sessionPage.getSpeakerName = function() {
+  const self = this;
+  const speaker = 'desc-speaker-name';
+
+  const sessionPromise = new Promise(function(resolve) {
+    const title = self.find(By.className(speaker)).getText();
+
+    resolve(title);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)